core/src/main/java/org/apache/struts2/StrutsConstants.java

```diff
@@ -41,12 +41,15 @@
     /** The URL extension to use to determine if the request is meant for a Struts action */
     public static final String STRUTS_ACTION_EXTENSION = "struts.action.extension";
 
-	/** Comma separated list of patterns (java.util.regex.Pattern) to be excluded from Struts2-processing */
```

Comment:
Would it be possible to separate those formatting and "fix typo" changes into one separate commit?

This would simplify the review.

Reply:
The "fix typo" changes were minimal enough that I figured it would be OK.  It might be possible to separate those changes out, but I'm pretty new to Git.  Presumably that would involve a new commit/push/pull request sequence (after trying to undo the formatting fixes) ?

Reply:
The following steps will do it.

```
# reset last commit
git reset HEAD~

# see your changes unstaged
git status

# partially add formatting and typo changes
# see for example https://johnkary.net/blog/git-add-p-the-most-powerful-git-feature-youre-not-using-yet/
git add -p .
git commit -m"Fix formatting and some typos"

# partially add main changes
git add -p .
git commit -m"WW-4971 Fix broken includes for non-UTF8 content"

# check that all wanted changes are included
git status

# log should contain two new commits
git log

# http://weiqingtoh.github.io/force-with-lease/
git push --force-with-lease

```

Reply:
Hint: If anything goes wrong or is unclear, don't do the push and instead repull from guthub.

Reply:
I wrote an instruction how to use GitHub, let me know if anything is unclear and I will be happy to improve it :)
https://struts.apache.org/submitting-patches.html#contributing-with-github

Reply:
Thank you for the helpful Git instructions, @sepe81 and @lukaszlenart.  The instructions on the Struts website seem to be pretty clear for "normal" contribution usage, and @sepe81 's instructions here seemed to work well for breaking up the changes into two sections (something well beyond my Git skills without such help).  
I ended up doing them in the reverse order he suggested (1st commit was for the proposed fix only, 2nd commit overlays typo fixes on top of that one), so hopefully that will be OK.   :)  :) 
Let me know if it worked out, if not I'll try again (this time around the 1st commit also includes some of the suggested changes from comments, hopefully it's a bit cleaner).